app/indexers/hyrax/indexers/resource_indexer.rb

```diff
@@ -39,7 +39,7 @@ def initialize(resource:)
       # @api public
       # @return [HashWithIndifferentAccess<Symbol, Object>]
       def to_solr
-        {
+        solr_hash = {
```

Comment:
This shouldn't be necessary

Reply:
ah nice catch, that was from the old way we were indexing resource.id